jupyterlab_bigquery/src/components/details_panel/dataset_details_panel.tsx

```diff
@@ -9,7 +9,7 @@ import LoadingPanel from '../loading_panel';
 import { DetailsPanel } from './details_panel';
 import { stylesheet } from 'typestyle';
 
-const localStyles = stylesheet({
+export const localStyles = stylesheet({
   body: {
     marginBottom: '24px',
     marginRight: '24px',
```

Comment:
I'm hoping to do some style refactoring to pull out some shared body/container styles in a later PR